src/javascript/app/pages/new_account.js

```diff
@@ -83,8 +83,8 @@ const NewAccount = (() => {
 
     const onUnload = () => {
         getElementById('footer').setVisibility(1);
-        $google_btn.off('click');
-        $login_btn.off('click');
+        if ($google_btn) $google_btn.off('click');
+        if ($login_btn) $login_btn.off('click');
```

Comment:
should we just move it to `off()` before it does `on()` on line 45? we don't really follow any strict convention for these

Reply:
oh yes, that's better